open3d conversionはビルドできるけど sbgisen_move_group_capabilitiesがビルドできない
```bash
Finished <<< open3d_conversions [17.5s]                                                                                                                                                       
Starting >>> sbgisen_move_group_capabilities                                                                                                                                                  
--- stderr: sbgisen_move_group_capabilities                                                                                                                                                   
CMake Deprecation Warning at /home/ubuntu/ros/install/open3d_vendor/lib/cmake/Open3D/Open3DConfig.cmake:18 (cmake_policy):                                                                    
  The OLD behavior for policy CMP0072 will be removed from a future version                                                                                                                   
  of CMake.                                                                                                                                                                                   
                                                                                                                                                                                              
  The cmake-policies(7) manual explains that the OLD behaviors of all                                                                                                                         
  policies are deprecated and that a policy should be set to OLD only under                                                                                                                   
  specific short-term circumstances.  Projects should be ported to the NEW                                                                                                                    
  behavior and not rely on setting a policy to OLD.                                                                                                                                           
Call Stack (most recent call first):                                                                                                                                                          
  /home/ubuntu/ros/install/open3d_conversions/share/open3d_conversions/cmake/ament_cmake_export_dependencies-extras.cmake:21 (find_package)                                                   
  /home/ubuntu/ros/install/open3d_conversions/share/open3d_conversions/cmake/open3d_conversionsConfig.cmake:41 (include)                                                                      
  /opt/ros/jazzy/share/ament_cmake_auto/cmake/ament_auto_find_build_dependencies.cmake:67 (find_package)                                                                                      
  CMakeLists.txt:7 (ament_auto_find_build_dependencies) 

CMake Deprecation Warning at /home/ubuntu/ros/install/open3d_vendor/lib/cmake/Open3D/Open3DConfig.cmake:18 (cmake_policy):                                                                    
  The OLD behavior for policy CMP0072 will be removed from a future version                                                                                                                   
  of CMake.                                                                                                                                                                                   

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.
Call Stack (most recent call first):
  /home/ubuntu/ros/install/open3d_vendor/share/open3d_vendor/cmake/open3d_vendor-extras.cmake:1 (find_package)
  /home/ubuntu/ros/install/open3d_vendor/share/open3d_vendor/cmake/open3d_vendorConfig.cmake:41 (include)
  /opt/ros/jazzy/share/ament_cmake_auto/cmake/ament_auto_find_build_dependencies.cmake:67 (find_package)
  CMakeLists.txt:7 (ament_auto_find_build_dependencies)


In this package, headers install destination is set to `include` by ament_auto_package. It is recommended to install `include/sbgisen_move_group_capabilities` instead and will be the default behavior of ament_auto_package from ROS 2 Kilted Kaiju. On distributions before Kilted, ament_auto_package behaves the same way when you use USE_SCOPED_HEADER_INSTALL_DIR option.
CMake Error at /home/ubuntu/ros/install/open3d_conversions/share/open3d_conversions/cmake/export_open3d_conversionsExport.cmake:61 (set_target_properties):
  The link interface of target "open3d_conversions::open3d_conversions"
  contains:

    pcl_msgs::pcl_msgs__rosidl_generator_c

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  /home/ubuntu/ros/install/open3d_conversions/share/open3d_conversions/cmake/ament_cmake_export_targets-extras.cmake:9 (include)
  /home/ubuntu/ros/install/open3d_conversions/share/open3d_conversions/cmake/open3d_conversionsConfig.cmake:41 (include)
  /opt/ros/jazzy/share/ament_cmake_auto/cmake/ament_auto_find_build_dependencies.cmake:67 (find_package)
  CMakeLists.txt:7 (ament_auto_find_build_dependencies)


CMake Generate step failed.  Build files cannot be regenerated correctly.
---
Failed   <<< sbgisen_move_group_capabilities [2.49s, exited with code 1]

Summary: 7 packages finished [2min 57s]
  1 package failed: sbgisen_move_group_capabilities

```

--- a/open3d_conversions/CMakeLists.txt
+++ b/open3d_conversions/CMakeLists.txt
@@ -7,15 +7,9 @@ if(NOT CMAKE_CXX_STANDARD)
 endif()
 
 # system dependencies
-find_package(ament_cmake REQUIRED)
-find_package(ament_cmake_ros REQUIRED)
+find_package(ament_cmake_auto REQUIRED)
 find_package(Eigen3 REQUIRED)
-
-find_package(open3d_vendor REQUIRED)
-find_package(rclcpp REQUIRED)
-find_package(rcpputils REQUIRED)
-find_package(sensor_msgs REQUIRED)
-find_package(pcl_conversions REQUIRED)
+ament_auto_find_build_dependencies()
 
 include_directories(
   include
@@ -23,44 +17,20 @@ include_directories(
   ${Open3D_INCLUDE_DIRS}
 )
 
-add_library(open3d_conversions src/open3d_conversions.cpp)
-target_include_directories(open3d_conversions PUBLIC
-  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
-  $<INSTALL_INTERFACE:include>)
-ament_target_dependencies(
-  open3d_conversions
-  "rclcpp"
-  "rcpputils"
-  "sensor_msgs"
-  "pcl_conversions"
-)
+ament_auto_add_library(open3d_conversions src/open3d_conversions.cpp)
 target_link_libraries(open3d_conversions ${Open3D_LIBRARIES})
 target_compile_definitions(open3d_conversions PUBLIC "OPEN3D_CONVERSIONS_BUILDING_LIBRARY")
-
-install(
-  DIRECTORY include/
-  DESTINATION include
-)
-install(
-  TARGETS open3d_conversions
-  EXPORT export_${PROJECT_NAME}
-  ARCHIVE DESTINATION lib
-  LIBRARY DESTINATION lib
-  RUNTIME DESTINATION bin
-  INCLUDES DESTINATION include
-)
 
 if(BUILD_TESTING)
   find_package(ament_cmake_gtest REQUIRED)
   find_package(ament_lint_auto REQUIRED)
   ament_add_gtest(test_open3d_conversions test/test_open3d_conversions.cpp)
-  target_link_libraries(test_open3d_conversions open3d_conversions ${Open3D_LIBRARIES})
+  target_link_libraries(
+    test_open3d_conversions
+    open3d_conversions
+    ${Open3D_LIBRARIES}
+  )
   ament_lint_auto_find_test_dependencies()
 endif()
 
-ament_export_include_directories(include)
-ament_export_libraries(open3d_conversions)
-ament_export_targets(export_${PROJECT_NAME})
-ament_export_dependencies(Open3D rclcpp)
-
-ament_package()
+ament_auto_package()

--- a/open3d_conversions/package.xml
+++ b/open3d_conversions/package.xml
@@ -13,7 +13,7 @@
 
   <author email="matnay17@gmail.com">Pranay Mathur</author>
   <author email="nkhedekar@nevada.unr.edu">Nikhil Khedekar</author>
-  <buildtool_depend>ament_cmake_ros</buildtool_depend>
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
   <depend>open3d_vendor</depend>
   <depend>rclcpp</depend>

